icu4c/source/i18n/unicode/numberrangeformatter.h

```diff
@@ -725,7 +727,9 @@ class U_I18N_API FormattedNumberRange : public UMemory, public FormattedValue {
     // Copydoc: this method is new in ICU 64
     /** @copydoc FormattedValue::nextPosition() */
     UBool nextPosition(ConstrainedFieldPosition& cfpos, UErrorCode& status) const U_OVERRIDE;
+#endif  /* U_HIDE_DRAFT_API */
```

Comment:
Yep, that is why we have the Travis conditionals test. Anyway, I did unhide the concrete override here and in a few other places. Also, though I had removed the guard conditionals around the ConstrainedFieldPosition class as a whole (since it is used by unguarded items), I added them to its methods. And oddly, I had to restore some @internal status tags to the private fields in the Precision class to avoid doxygen complaints (not sure why it cares about private items). Anyway, all tests are passing now.